### PR TITLE
links key support

### PR DESCRIPTION
**Pull request by onumossn**
https://github.com/Netflix/fast_jsonapi/issues/94

### CODE REVIEW

**Comment by shishirmk** [comment]
@onumossn Do you mind adding documentation about how some would enable the links?

**Comment by onumossn** [comment]
Is documentation through example be acceptable since there does not seem to be a section for API docs and the examples seem self-explanatory?

File: lib/fast_jsonapi/object_serializer.rb [hunk 1]
```diff
@@ -47,6 +47,7 @@ def serializable_hash
     def hash_for_one_record
       serializable_hash = { data: nil }
       serializable_hash[:meta] = @meta if @meta.present?
+      serializable_hash[:links] = @links if @links.present?
```
[the remark this answers is not in the document]
Comment (onumossn) [reply lib/fast_jsonapi/object_serializer.rb]: `hash_for_collection` does not seem to use `hash_for_one_record` in any way from my understanding, so I don't think so. It should behave the same way as `meta`.

[the remark this answers is not in the document]
Comment (shishirmk) [reply lib/fast_jsonapi/object_serializer.rb]: Ya you are right. I am going to merge this in

**Comment by shishirmk** [comment]
@onumossn thank you for making the pull request

**Pull request merged**